.circleci/config.yml

```diff
@@ -24,6 +24,12 @@ workflows:
           python_version: << pipeline.parameters.python_version >>
           markers: integration
 
+  publish-docs:
+    when:
+      equal: [<< pipeline.git.branch >>, develop]
+    jobs:
+      - build-and-push-docs
+
```

Comment:
I don't think we should always publish docs when merging to develop. We need to make a distinction between general changes that don't apply to a specific version (like correction, clarifications, formatting,...) and updates for unreleased code.

Reply:
Then what would be the ruleset we use to determine if we should publish docs?

Reply:
On second thought, If we use versioning we can always publish. But it's not going to show up on the website, since it's not tied to a version.